packages/common/src/ui/layout/tables/components/Cells/CurrencyInputCell/CurrencyInputCell.tsx

```diff
@@ -23,7 +23,7 @@ export const CurrencyInputCell = <T extends DomainObject>({
     <CurrencyInput
       autoFocus={autoFocus}
       maxWidth={column.width}
-      value={buffer}
+      defaultValue={buffer}
```

Comment:
this does look pretty bad with `xof` in it - any thoughts about having the currency symbol outside the input?

<img width="145" alt="Screen Shot 2022-03-01 at 4 49 06 PM" src="https://user-images.githubusercontent.com/9192912/156101411-40c560e6-6447-4041-834a-0299e37deaba.png">

<img width="137" alt="Screen Shot 2022-03-01 at 4 49 55 PM" src="https://user-images.githubusercontent.com/9192912/156101416-aa066db0-6ebc-4693-84a4-19dc004e5ce6.png">

<img width="152" alt="Screen Shot 2022-03-01 at 4 50 12 PM" src="https://user-images.githubusercontent.com/9192912/156101425-a72c9d35-9b6b-4340-82c4-f5c70df7cbb7.png">

Looks ok with a numeric input too..

<img width="219" alt="Screen Shot 2022-03-01 at 4 45 39 PM" src="https://user-images.githubusercontent.com/9192912/156101097-e26c8568-177b-44b2-8311-31bf4dc26079.png">

<img width="215" alt="Screen Shot 2022-03-01 at 4 46 41 PM" src="https://user-images.githubusercontent.com/9192912/156101114-ff7d6e49-018e-400c-ac75-0bf35cec544b.png">

Reply:
I've no preference either way. Do you have a branch with the changes you have in the pics? Can we just commit that 🤷